thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
```

Comment:
Can we please rename this component to "anomaly-graph.js" `../thirdeye-frontend/app/pods/components/anomaly-graph.js`? Conventionally, Ember creates 3 files when one creates a component through Ember CLI. `.../components/component-name.js`, `.../templates/components/component-name.hbs`, and `../tests/integration/components/component-name.js`. Would be good to be consistent with Ember naming/structure conventions :)